profiles/disa_stig-rhel7-baseline/libraries/profile_helper/audit.rb

```diff
@@ -0,0 +1,71 @@
+def check_paths(paths, permissions)
```

Comment:
With the new auditd resource, I think a lot of this will get cleaned up through the usage of FilterTables and the "where" accessor. I'll work on getting a good example that has the same logic as this code.